src/eos-autoupdater.c

```diff
@@ -66,6 +66,7 @@ static const char *UPDATE_STAMP_NAME = "eos-updater-stamp";
 static const char *CONFIG_FILE_PATH = SYSCONFDIR "/" PACKAGE "/eos-autoupdater.conf";
 static const char *OLD_CONFIG_FILE_PATH = SYSCONFDIR "/eos-updater.conf";
 static const char *STATIC_CONFIG_FILE_PATH = PKGDATADIR "/eos-autoupdater.conf";
+static const char *VAR_CONFIG_FILE_PATH = LOCALSTATEDIR "/local/" PACKAGE "/eos-autoupdater.conf";
```

Comment:
This should be /usr/local rather than /var - the fact that ends up in /var is an ostree implementation deal.